dags/elsevier/metadata_parser.py

```diff
@@ -40,26 +44,36 @@ def __init__(self, doi, file_path) -> None:
                 extraction_function=self._get_license,
                 required=True,
             ),
-            # CustomExtractor(
-            #     destination="local_files",
-            #     extraction_function=self._get_local_files,
-            #     # required=True,
-            # ),
+            CustomExtractor(
+                destination="files",
+                extraction_function=self._get_local_files,
```

Comment:
aren't we ignoring local files?

Reply:
yes, but I implemented them because we will need them later, so we could attach files to the record. It is easier to implement this functionality now, since paths are parsed from XML